lighthouse-cli/printer.ts

```diff
@@ -8,8 +8,8 @@
 /**
  * An enumeration of acceptable output modes:
  *   'json': JSON formatted results
- *   'html': An HTML report
- *   'domhtml': An HTML report rendered client-side with DOM elements
+ *   'html': An HTML report rendered client-side with DOM elements
+ *   'domhtml': Alias for 'html' report.
```

Comment:
can you also tweak the filename for domhtml files to be just `.html`? That's in bin.ts